fmriprep/interfaces/images.py

```diff
@@ -232,7 +232,7 @@ def _run_interface(self, runtime):
         target_span = target_shape * target_zooms
 
         zooms = np.array(reoriented.header.get_zooms()[:3])
-        shape = np.array(reoriented.shape)
+        shape = np.array(reoriented.shape[:3])
```

Comment:
If we don't need to resample 4D images, then the output of this node will still be a 4D image. Do the downstream nodes handle this okay?

Reply:
This change is actually a downstream artifact of the change in line 166.

I'm testing this out right now, so far so good.

Reply:
It went through the whole execution with ``--anat-only`` on the problematic dataset :+1:

Reply:
Sounds good.